oneview/resource_server_profile_template.go

```diff
@@ -1541,12 +1533,11 @@ func resourceServerProfileTemplateCreate(d *schema.ResourceData, meta interface{
 		for _, raw := range rawFirmware {
 			firmwareItem := raw.(map[string]interface{})
 			firmware = ov.FirmwareOption{
-				ForceInstallFirmware:     firmwareItem["force_install_firmware"].(bool),
-				FirmwareBaselineUri:      utils.NewNstring(firmwareItem["firmware_baseline_uri"].(string)),
-				ManageFirmware:           firmwareItem["manage_firmware"].(bool),
-				FirmwareInstallType:      firmwareItem["firmware_install_type"].(string),
-				FirmwareActivationType:   firmwareItem["firmware_activation_type"].(string),
-				FirmwareScheduleDateTime: firmwareItem["firmware_schedule_date_time"].(string),
+				ForceInstallFirmware:   firmwareItem["force_install_firmware"].(bool),
+				FirmwareBaselineUri:    utils.NewNstring(firmwareItem["firmware_baseline_uri"].(string)),
```

Comment:
why we are not taking care of compliance_control property? its there in the schema